src/reference/asciidoc/whats-new.adoc

```diff
@@ -23,3 +23,8 @@ See its JavaDocs and <<./graph.adoc#integration-graph,Integration Graph>> for mo
 
 [[x5.3-general]]
 === General Changes
+=======
+[[x5.3-general]]
+=== General Changes
```

Comment:
Don't you find this as a duplication of already existing text?

Reply:
Had to manually merge it. Ooops